packages/components/src/InputDate/InputDate.tsx

```diff
@@ -69,22 +69,24 @@ export function InputDate(inputProps: InputDateProps) {
         value && formFieldActionsRef.current?.setValue(value);
 
         return (
-          <FormField
-            {...newActivatorProps}
-            {...inputProps}
-            value={value}
-            onChange={(_, event) => onChange && onChange(event)}
-            onBlur={() => {
-              inputProps.onBlur && inputProps.onBlur();
-              activatorProps.onBlur && activatorProps.onBlur();
-            }}
-            onFocus={() => {
-              inputProps.onFocus && inputProps.onFocus();
-              activatorProps.onFocus && activatorProps.onFocus();
-            }}
-            actionsRef={formFieldActionsRef}
-            suffix={suffix}
-          />
+          <div onClick={onClick}>
```

Comment:
i think this merits a comment on the code or github as to why this exist

Reply:
fair. I've added something lmk if you think that's descriptive enough or have any suggestions!

Reply:
I like it!